build_tools/littlecheck.py

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env python
 
+# SPDX-FileCopyrightText: © 2019 fish-shell contributors
+#
+# SPDX-License-Identifier: GPL-2.0-only
```

Comment:
This is incorrect - littlecheck is imported [from here](https://github.com/ridiculousfish/littlecheck) and licensed as "public domain" (the intention there obviously being "do whatever you want with it"), by us.

Reply:
And since it's vendored from another repo, the typical way of updating it is the copy the file over, so this would be broken again.

Reply:
Got it.

There’s three practical solutions I see:
1. make that repository REUSE compliant as well, so the file would already carry the right license and copyright info – optimal, but more work
2. move the (right) SPDX/REUSE tags into a new file `build_tools/littlecheck.py.license`, so that info would apply to the file even if it’s later updated – has its downsides, but might be a good compromise here
3. add the right info on every update of the file – annoying

Reply:
I created https://github.com/ridiculousfish/littlecheck/pull/11 in order to tackle this issue with the option 1) solution.

If that is not desirable, I’ll change it accordingly.